packages/sit-onyx/src/components/OnyxVisuallyHidden/types.ts

```diff
@@ -0,0 +1,3 @@
+export type OnyxVisuallyHiddenProps = {
+  is?: string;
```

Comment:
Would prefer to make it required so its more explicit

Reply:
Usually there should not be a need to change the element.